README.md

```diff
@@ -29,12 +29,16 @@ The development of TTG was motivated by _irregular_ scientific applications like
 
 int main(int argc, char *argv[]) {
   ttg::initialize(argc, argv);
-
+  // a simple template task
   auto tt = ttg::make_tt([]() { std::cout << "Hello, World!\n"; });
 
+  // make sure the graph is connected (simple for a single node)
```

Comment:
this is not sufficient to understand why the graph needs to be connected, or why it is simple for a single node.